iothub/device/tests/ConnectionString/DeviceClientConnectionStringTests.cs

```diff
@@ -280,14 +316,13 @@ public void DeviceClientIotHubConnectionStringBuilderTest()
             Assert.IsNotNull(iotHubConnectionStringBuilder.SharedAccessSignature);
             Assert.IsTrue(iotHubConnectionStringBuilder.AuthenticationMethod is DeviceAuthenticationWithToken);
 
-            try
-            {
-                iotHubConnectionStringBuilder.HostName = "adshgfvyregferuehfiuehr";
-                Assert.Fail("Expected FormatException");
-            }
-            catch (FormatException)
-            {               
-            }
+            iotHubConnectionStringBuilder.HostName = "adshgfvyregferuehfiuehr";
+            iotHubConnectionStringBuilder.AuthenticationMethod = new DeviceAuthenticationWithRegistrySymmetricKey("Device1-.+%_#*?!(),=@;$'", "dGVzdFN0cmluZzE=");
```

Comment:
Best pull these out as local constants to use in validation (below)

Reply:
done